packages/api/src/controllers/dashboard.ts

```diff
@@ -1,24 +1,58 @@
-import { differenceBy, uniq } from 'lodash';
+import { differenceBy, groupBy, uniq } from 'lodash';
 import { z } from 'zod';
 
-import { DashboardWithoutIdSchema, Tile } from '@/common/commonTypes';
+import { DashboardWithoutIdSchema } from '@/common/commonTypes';
 import type { ObjectId } from '@/models';
 import Alert from '@/models/alert';
 import Dashboard from '@/models/dashboard';
-import { tagsSchema } from '@/utils/zod';
 
 export async function getDashboards(teamId: ObjectId) {
-  const dashboards = await Dashboard.find({
+  const _dashboards = await Dashboard.find({
     team: teamId,
   });
+
+  const alertsByTileId = groupBy(
+    await Alert.find({
+      dashboard: { $in: _dashboards.map(d => d._id) },
+      source: 'tile',
+    }),
+    'tileId',
+  );
+
+  const dashboards = _dashboards
+    .map(d => d.toJSON())
+    .map(d => ({
+      ...d,
+      tiles: d.tiles.map(t => ({
+        ...t,
+        config: { ...t.config, alert: alertsByTileId[t.id]?.[0] },
```

Comment:
any reason we want to add `alert` field to `config` instead of adding it to the top level?